Formula/marcli.rb

```diff
@@ -0,0 +1,28 @@
+class Marcli < Formula
+  desc "Parse MARC (ISO 2709) files"
+  homepage "https://github.com/hectorcorrea/marcli"
+  url "https://github.com/hectorcorrea/marcli/archive/refs/tags/1.0.1.tar.gz"
+  sha256 "abea9ae6a7cd2d6874ac71d0aa429b9eab367daa4f301161a0671f85608216cf"
+  license "MIT"
+  head "https://github.com/hectorcorrea/marcli"
+
+  depends_on "go" => :build
+
+  resource "testdata" do
+    url "https://raw.githubusercontent.com/hectorcorrea/marcli/main/data/test_1a.mrc"
```

Comment:
```suggestion
    url "https://raw.githubusercontent.com/hectorcorrea/marcli/HEAD/data/test_1a.mrc"
```

Reply:
Nice catch. We should probably pin this to a specific commit, so we don't see checksum failures from this.

Reply:
Updated my suggestion. I've used the full commit hash rather than the abbreviated one, feel free to edit my suggestion if required.

Reply:
Ah OK that makes perfect sense. GitHub question—I've been squashing commits locally in git and then `git push --force` to the branch. Is it possible to accept this commit suggestion and then squash on GitHub? Sorry if this is a noob question, just hoping to find an easier way.

Reply:
No problem, I don't think squashing is possible 😅

Reply:
OK, incorporated this change!